### DISCUSSION TITLE
How to delete short url created

### DISCUSSION THREAD

**Discussion by uluumbch** [ash-jc-allen/short-url#130] [Q&A] [answered]
i had deleted shrot url created in my database, but after deleted, i still can visit shorted url. did i get wrong? hwo to delete then?

**Comment by ash-jc-allen** [comment, marked as answer]
Hey @uluumbch! Would you please be able to show me the snippets of code where you're creating and deleting the short URLs? I'll do a bit of looking into it for you 🙂
